bin/dashboard-server

```diff
@@ -2,7 +2,7 @@
 require_relative '../deployment'
 
 def main
-  rerun = "rerun -p '**/*.{rb,ru,yml}' -d'#{deploy_dir('lib')},#{deploy_dir('shared/middleware')},#{pegasus_dir}' --" if CDO.dashboard_enable_pegasus
+  rerun = "rerun -p '**/*.{rb,ru,yml}' -d '#{deploy_dir('lib')},#{deploy_dir('shared/middleware')},#{pegasus_dir}' -i '**/migrations/*.rb' --" if CDO.dashboard_enable_pegasus
```

Comment:
What does this do? For my use case, I'd want `dashboard-server` not to initiate any migrations.